src/app/pets/pet-edit/pet-edit.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="row">
+    <div class="col-xs-12">
+        <form [formGroup]="petForm" (ngSubmit)="onSubmit()">
+            <div class="row">
+                <div class="col-xs-12">
+                    <label for="name">Name</label>
+                    <input type="text" id="name" class="form-control" formControlName="name">
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-xs-12">
+                    <label for="breed">Breed</label>
+                    <input type="text" id="breed" class="form-control" formControlName="breed">
+                </div>
+            </div>
+            <hr>
+            <div class="row">
+                <div class="col-xs-12">
+                    <button type="submit" class="btn btn-success" [disabled]="!petForm.valid">{{ editMode ? 'Update' : 'Add' }}</button> | 
+                    @if (editMode) {
```

Comment:
this is ok but its an angular 17 feature, just make sure you also know how to do this using ngtemplates and ngIf